modules/tinymce/changelog.txt

```diff
@@ -10,6 +10,7 @@ Version 5.4.0 (TBD)
     Fixed handling of mixed-case icon names by having the silver theme downcase them before icon retrieval #TINY-3854
     Fixed leading and trailing spaces lost when using `editor.selection.getContent({ format: 'text' })` #TINY-5986
     Fixed an issue where removing formatting in a table cell would cause Internet Explorer 11 to scroll to the end of the table #TINY-6049
+    Added `StyleSheetLoader` module to public API, as it was already part of `DOMUtils` #TINY-6100
```

Comment:
This should be at the top along with the other "Added" items.

Reply:
Fixed.